pipelines/ApPipe.yaml

```
@@ -24,10 +24,16 @@ parameters:
   diffWarp: goodSeeingDiff_warpedExp
   diffMatch: goodSeeingDiff_matchedExp
   assocSrc: goodSeeingDiff_assocDiaSrc
+  templateExp: goodSeeingDiff_templateExp
   # TODO: end DM-30210 workaround
 tasks:
+  retrieveTemplate:  # For multi-tract difference imaging
+    class: lsst.ip.diffim.getTemplate.GetMultiTractCoaddTemplateTask
+    config:
+      connections.coaddExposures: goodSeeingCoadd
```

Didn't notice this before:
```suggestion
      connections.coaddExposures: parameters.template
```
(Perhaps the parameter should be renamed, but it might be better to do that on a different ticket.)